src/views/AddItem.jsx

```diff
@@ -19,6 +19,9 @@ export function AddItem({ listToken }) {
 	// Notify when adding item is unsuccessful
 	const notifyFailedRequest = () => toast.error('Failed to add item');
 
+	// Notify if the item already exists in db
+	const notifyInList = () => toast.error('Already in your list');
+
```

Comment:
I love our team's love of react-hot-toast! I'm starting to think if it might be useful for us to have a reusable file of all of our toast notifications that we can import to our frontend components. But that may not be as efficient as I think it would be. Overall, I'm excited for our use of toast notifications and can't wait to style our app down the line (and maybe [style the toasts](https://react-hot-toast.com/docs/styling) too 👀 )

Reply:
@alex-andria Yess, styling the toasts is something I look forward to! Good point about creating a separate hot toast file.